bindings/c/include/vw/experimental/exports.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+#pragma once
+
+#if defined _WIN32 || defined __CYGWIN__
+  #ifdef VWDLL_EXPORTS
+    #ifdef __GNUC__
+      #define VW_DLL_PUBLIC __attribute__((dllexport))
+    #else
+      #define VW_DLL_PUBLIC __declspec(dllexport)
+    #endif
+  #else
+    #ifdef __GNUC__
+      #define VW_DLL_PUBLIC __attribute__((dllimport))
+    #else
+      #define VW_DLL_PUBLIC __declspec(dllimport)
+    #endif
+  #endif
+  #define VW_DLL_LOCAL
+#else
+  #if __GNUC__ >= 4
+    #define VW_DLL_PUBLIC __attribute__((visibility("default")))
+    #define VW_DLL_LOCAL __attribute__((visibility("hidden")))
+  #else
+    #define VW_DLL_PUBLIC
+    #define VW_DLL_LOCAL
+  #endif
+#endif
+
+#ifdef __cplusplus
```

Comment:
apart from lines 31-35 the rest of the file is the same as `vwvis.h` right? Would it make sense to add the lines 31-35 to the `vwvis.h` file and include it where needed, or copy the file like we do `error_data.h` in order to avoid code duplication?

Reply:
Good point, will use the same process as error_data